RecoBTag/DeepFlavour/python/DeepFlavourTagInfos_cfi.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
+
+pfDeepFlavourTagInfos = cms.EDProducer(
+	'DeepFlavourTagInfoProducer',
+  jets = cms.InputTag("ak4PFJetsCHS"),
+  vertices = cms.InputTag("offlinePrimaryVertices"),
+  secondary_vertices = cms.InputTag("inclusiveCandidateSecondaryVertices"),
+  shallow_tag_infos = cms.InputTag('pfDeepCSVTagInfos'),
+  puppi_value_map = cms.InputTag('puppi'),
+  vertexAssociator = cms.InputTag('primaryVertexAssociation','original'),
```

Comment:
any reason this was left in camel cap, considering everything else is underscore-separated

Reply:
no reason, taken from other CMSSW module, will change for consistency

Reply:
changed with https://github.com/cms-sw/cmssw/pull/19893/commits/c2dc5c01427a4ea10ed8cba83f44382c986ebb46